tests/by-util/test_env.rs

```diff
@@ -445,6 +445,7 @@ macro_rules! compare_with_gnu {
 }
 
 #[test]
+#[allow(clippy::cognitive_complexity)] // Ignore clippy lint of too long function sign
```

Comment:
Still necessary with the higher limit?

Reply:
Of course, it's actually more than 25

But I think the signature of this function expresses a very clear meaning, so it's best to add #[allow(clippy::cognitive_complexity)]

Reply:
Here's the log before, it's about 71 length

```
error: the function has a cognitive complexity of (71/24)
   --> tests/by-util/test_env.rs:448:4
    |
448 | fn test_env_with_gnu_reference_parsing_errors() {
    |    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = help: you could split it up into multiple smaller functions
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#cognitive_complexity
    = note: `-D clippy::cognitive-complexity` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(clippy::cognitive_complexity)]`
```

Reply:
Alright, I was just looking at the diff. Looks like the macro is introducing a lot of complexity according to clippy 😄